batch2/batch/utils.py

```diff
@@ -18,10 +18,14 @@ def parse_cpu(cpu_string):
         number = float(match.group(1))
         if match.group(2) == 'm':
             number /= 1000
-        return round(number, 3)
+        return int(number * 1000)  # mCPU
     return None
 
 
+def mcpu_to_cpu(mcpu):
+    return round(mcpu / 1000, 3)
```

Comment:
I think you can just divide by 1000 in context.  Any reader should know what the milli- prefix means.  And the round is unnecessary because mcpu is an integer.